tests/trig/eval/manifest.ttl

```diff
@@ -11,7 +11,16 @@ PREFIX rdft:   <http://www.w3.org/ns/rdftest#>
 PREFIX trs:    <https://w3c.github.io/rdf-star/tests/trig/eval#>
 
 trs:manifest  rdf:type mf:Manifest ;
-    rdfs:label "TriG-star Evaluation Tests" ;
+    rdfs:label "TriG-star Evaluation Tests"@en ;
+    rdfs:label "La suite des tests pour TriG-star"@fr;
+	 rdfs:label "Conjunto de pruebas para Trig-star"@es;
+	dct:issued "2021-06-21"^^xsd:date ; #not sure if this was the first release date
+	dct:creator [ foaf:name "Gregg Kellog" ] ; #maybe add foaf:homepage
+	dct:creator [ foaf:name "Andy Seaborne" ];
```

Comment:
I don't think we should name people. The output is from the community. creator compared to contributor are not so easily distinguished.

Reply:
OK. I'll just mention the name of the group using `dct:creator` and remove the `dct:contributor`

Reply:
Pushed a new version